common/utils.js

```diff
@@ -785,6 +785,40 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function estimateITSFee(chain, destinationChain, env, eventType, gasValue, _axelar) {
+    if (env === 'devnet-amplifier') {
+        printInfo('Using default ITS fee: 0');
+        return 0;
+    }
+
+    if (isValidNumber(gasValue)) {
+        return scaleGasValue(chain, gasValue);
+    }
+
+    const url = `${_axelar?.axelarscanApi}/gmp/estimateITSFee`;
+
+    const payload = {
+        sourceChain: chain.axelarId,
+        destinationChain: destinationChain,
+        event: eventType,
+    };
+
+    const res = await httpPost(url, payload);
+
+    if (res.error) {
+        throw new Error(`Error querying gas amount: ${res.error}`);
+    }
+    return res;
+}
+
+function scaleGasValue(chain, gasValue) {
+    if (typeof chain.gasScalingFactor === 'number') {
```

Comment:
I'd create a separate task to change the behaviour of gas value override. Instead of passing in raw wei amount, let user pass in a decimal value, 1.5 eth etc. and then scale it per chain based on it's native token decimals to be more user friendly